Makefile

```diff
@@ -39,7 +39,8 @@ BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
 BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 
-CRD_OPTIONS ?= "crd:generateEmbeddedObjectMeta=true,maxDescLen=200"
+# kubectl apply does not work on large CRDs.
+CRD_OPTIONS ?= "crd:generateEmbeddedObjectMeta=true,maxDescLen=0"
```

Comment:
I don't think we want to publish CRDs with no descriptions on fields?

Reply:
Unfortunately the `kubectl apply` on the CRD will not work (https://github.com/open-telemetry/opentelemetry-operator?tab=readme-ov-file#getting-started).

It will break installations, maybe helm, gitops and e2e tests?  

I can take a look at enabling descriptions in a follow-up PR. There we will better see what the implications are. Please book a follow-up ticket if you think we should have descriptions.

Reply:
[I think a server side apply actually does work for these](https://kubernetes.io/docs/reference/using-api/server-side-apply/#comparison-with-client-side-apply)